services/movie_session.py

```diff
@@ -0,0 +1,49 @@
+from django.db.models import QuerySet
+from django.utils.datetime_safe import datetime
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+    movie_show_time: datetime,
+    movie_id: int,
+    cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id),
+        movie=Movie.objects.get(id=movie_id)
```

Comment:
Consider handling the case where the CinemaHall or Movie with the given ID does not exist by using get_object_or_404 or handling the DoesNotExist exception to prevent potential errors.